### PR TITLE
FMD-37: add dev environment

### PR DESCRIPTION
**Pull request by gidsg**
### Change description
Add dev environment option so it can be deployed to manually

- [ ] Unit tests and other appropriate tests added or updated
- [ ] README and other documentation has been updated / added (if needed)
- [x] Commit messages are meaningful and follow good commit message guidelines (e.g. "FS-XXXX: Add margin to nav items preventing overlapping of logo")

### How to test
Have tested by deploying from this branch in: https://github.com/communitiesuk/funding-service-design-post-award-data-store/actions/runs/6497519823/job/17646812805

### CODE REVIEW

**Review by cyrusdobbs — CHANGES_REQUESTED** [review]
I feel as if it could be useful for some documentation to accompany these changes so it's clear how to access the data-store when it's hosted. Either in README or https://github.com/communitiesuk/funding-service-design-post-award-data-store/blob/main/scripts/ssm-conn.sh ?

**Comment by gidsg** [comment]
> I feel as if it could be useful for some documentation to accompany these changes so it's clear how to access the data-store when it's hosted. Either in README or https://github.com/communitiesuk/funding-service-design-post-award-data-store/blob/main/scripts/ssm-conn.sh ?

Added some documentation for this in eb29724

File: README.md [hunk 1]
```diff
@@ -142,3 +142,11 @@ App should be available at `http://localhost:8080`
...
 ## Deployment
 `main` branch is continuously deployed to AWS, see [.github/workflows/deploy.yml](.github/workflows/deploy.yml)
+
+## Accessing on AWS
+As it is deployed as a Backend Service on AWS the service is not publicly accessible.
+We have written a script to allow you to tunnel from your local machine to the environment
+* You will need to be granted AWS access with aws-vault setup as a pre-requisite (ask another developer how to gain access and set these up)
+* Use AWS vault to exec into the account for the environment you would like to access
+* Run: `./scripts/ssm-conn.sh 9999`
+* This should expose the remote environment at `http://localhost:9999`
```
Comment (cyrusdobbs) [inline README.md:152]: Can this include that the default params will work for the test and dev environments but for prod you will have to pass modified params the to include the `prod` suffix?

Comment (gidsg) [reply README.md:152]: Yes, there is a different command for prod but I think maybe that belongs better in a Confluence doc as we probably want to wrap more process around people doing stuff in production?

**Review by cyrusdobbs — APPROVED** [review]
LGTM

**Pull request merged**